lib/ansible/modules/windows/win_domain.py

```diff
@@ -71,6 +72,26 @@
     type: str
     choices: [ Win2003, Win2008, Win2008R2, Win2012, Win2012R2, WinThreshold ]
     version_added: '2.8'
+  parent_domain_name:
+    description:
+    - Specifies the parent domain while creating a new domain in a forest.
+    - Has to be the parent domain's fqdn
+    - It can't be used with forest mode as we don't create a new forest.
+    type: str
+    version_added: '2.9'
+  parent_domain_user:
+    description:
+    - Credential of the parent domain.
+    - The user has to be entreprise administrator to create a new subdomain.
+    - Required if parent_domain_name is defined.
+    type: str
+    version_added: '2.9'
+  parent_domain_password:
+    description:
+    - Password of the parent domain's entreprise administrator.
+    - Required if parent_domain_name is defined.
```

Comment:
Please look [here ](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation)to make the reference for the parameter